src/loaders/svg/tvgSvgUtil.cpp

```diff
@@ -263,7 +263,9 @@ string svgUtilURLDecode(const char *src)
 {
     if (!src) return nullptr;
 
-    auto length = strlen(src);
+    const char* x = src;
```

Comment:
How about `auto` instead of const char*?

Reply:
sorry, I forgot about that... changed